### PR TITLE
Remove data fields from adviser message

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
/unhold

PTAL

File: thoth/messaging/adviser_trigger.py [hunk 1]
```diff
@@ -55,7 +50,7 @@ class MessageContents(BaseMessageContents):  # type: ignore
         github_base_repo_url = attr.ib(type=Optional[str], default=None)
         re_run_adviser_id = attr.ib(type=Optional[str], default=None)
         source_type = attr.ib(type=Optional[str], default=None)
-        version = attr.ib(type=str, default="v1", init=False)
+        version = attr.ib(type=str, default="v2", init=False)
```
Comment (pacospace) [inline thoth/messaging/adviser_trigger.py:53]: this requires changes in investigator to be able to read this version I think. cc @KPostOffice

**Review by harshad16 — COMMENTED** [review]
/lgtm

File: thoth/messaging/adviser_trigger.py (hunk 1 above)
Comment (pacospace) [reply thoth/messaging/adviser_trigger.py:53]: https://github.com/thoth-station/investigator/blob/b51e4acacc2f2e2bda80a65527853ac1328a7e3d/thoth/investigator/adviser_trigger/investigate_adviser_trigger.py#L37

Comment (fridex) [reply thoth/messaging/adviser_trigger.py:53]: Anything else besides https://github.com/thoth-station/investigator/pull/445/ is needed?

Comment (fridex) [reply thoth/messaging/adviser_trigger.py:53]: Adjusted in https://github.com/thoth-station/investigator/pull/445/

Comment (pacospace) [reply thoth/messaging/adviser_trigger.py:53]: > Anything else besides [thoth-station/investigator#445](https://github.com/thoth-station/investigator/pull/445) is needed?

Not that comes to my mind :)

**Review by pacospace — COMMENTED** [review]
Should we remove them also from https://github.com/thoth-station/messaging/blob/master/thoth/messaging/adviser_re_run.py

and https://github.com/thoth-station/messaging/blob/master/thoth/messaging/provenance_checker_trigger.py

based on https://github.com/thoth-station/investigator/pull/445

**Comment by pacospace** [comment]
cc @fridex

**Comment by fridex** [comment]
> Should we remove them also from https://github.com/thoth-station/messaging/blob/master/thoth/messaging/adviser_re_run.py

Done in 85cb812

> and https://github.com/thoth-station/messaging/blob/master/thoth/messaging/provenance_checker_trigger.py

Done in 30cdad4

Will adjust investigator accrodingly. PTAL.

**Comment by pacospace** [comment]
Thanks!!

**Pull request merged**